code/modules/clothing/head/armor.dm

```diff
@@ -504,3 +504,21 @@
 		bio = 0,
 		rad = 0
 	)
+
+/obj/item/clothing/head/armor/helmet/crusader
+	name = "crusader helmet"
+	desc = "May God guide you."
+	icon_state = "crusader_hemet"
+	item_state = "crusader_hemet"
+	flags_inv = HIDEEARS|HIDEEYES|BLOCKHAIR
+	matter = list(MATERIAL_BIOMATTER = 15, MATERIAL_PLASTEEL = 5, MATERIAL_STEEL = 5, MATERIAL_GOLD = 1)
+	armor = list(
+		melee = 70,
+		bullet = 50,
```

Comment:
Same changes to helmet as to the main armor.
```suggestion
		bullet = 30,
```

Reply:
Done.